testsuite/MDAnalysisTests/coordinates/test_coordinates.py

```diff
@@ -123,12 +123,12 @@ def test_frame_numbering(self):
 
     def test_frame(self):
         self.trajectory[0]
-        coord0 = self.universe.atoms.coordinates().copy()
+        coord0 = self.universe.atoms.positions.copy()
```

Comment:
Is it ever necessary to call ``.copy()`` on the array we return with ``ag.positions``? Is it not always a copy?